### PR TITLE
Merge branch unit-09_mvc into unit-09

### PR DESCRIPTION
**Pull request by pipalalala**
Tests for custom model mappers were made

### CODE REVIEW

File: Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+        [Test]
+        public void GetBookInfoModel_WhenBookProfileIsNull_ShouldThrowArgumentNullException()
+        {
+            // Assert
+            Assert.Throws<ArgumentNullException>(() => bookModelMapper.GetBookInfoModel(EmptyBook));
+        }
+
+        [Test]
+        public void GetBookInfoModel_ShouldBookInfoModelFieldsMatchBookProfileFields()
```
Comment (aliaksanderkazlou) [inline Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs]: Non English sentence.

File: Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        {
+            // Arrange
+            BookInfoModel bookInfoModel = null;
+
+            // Act
+            bookInfoModel = bookModelMapper.GetBookInfoModel(bookProfile);
+
+            // Assert
+            Assert.AreEqual(bookInfoModel.Id, bookProfile.Id, "IDs not equal");
```
Comment (aliaksanderkazlou) [inline Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs]: Expected value should be passed as first parameter. Here expected value is bookProfile.id and it's passed as second parameter.

File: Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        public void GetBookFullInfoModel_ShouldMatchResultBookFieldsAndBookFieldsAndUserNameMatchNull()
+        {
+            // Arrange
+            BookFullInfoModel resultBook = null;
+
+            // Act
+            resultBook = bookModelMapper.GetBookFullInfoModel(book);
+
+            // Assert
+            Assert.AreEqual(resultBook.Id, book.Id, "IDs not equal");
```
Comment (UladzislauBiarnatskiGodel) [inline Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs]: Remove comments.

File: Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+        [Test]
+        public void GetBookFullInfoModel_WhenBookProfileIsNull_ShouldThrowArgumentNullException()
+        {
+            // Assert
+            Assert.Throws<ArgumentNullException>(() => bookModelMapper.GetBookFullInfoModel(EmptyBook));
+        }
+
+        [Test]
+        public void GetBookFullInfoModel_ShouldMatchResultBookFieldsAndBookFieldsAndUserNameMatchNull()
```
Comment (UladzislauBiarnatskiGodel) [inline Lab08.MVC/Lab08.MVC.Tests/Mappers/BookModelMapperTests.cs]: GetBookFullInfoModel_ShouldMapAllFields - it's better to use that name.

**Pull request merged**